src/pages/[networkType]/garden/[gardenAddress]/collateral.tsx

```diff
@@ -0,0 +1,3 @@
+import StakeManagement from '@/components/Garden/Stake/StakeManagement'
```

Comment:
From my understanding, I believe we are not getting the URL params right for all these next.js files.

Reply:
I didn't really understand what you meant here.

Are you asking how next.js will come to know about the `networkType` and `gardenAddress` from the URL? If so, that will be automatically resolved since the `[networkType]` and `[gardenAddress]` are dynamic parameters.